File: src/main/java/at/hrechny/predictionsbot/connector/apifootball/model/Status.java
package at.hrechny.predictionsbot.connector.apifootball.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

@Data
public class Status {

  @JsonProperty("long")
  private String statusLong;

  @JsonProperty("short")
  private FixtureStatusEnum status;

  private Integer elapsed;

}

File: src/main/kotlin/at/hrechny/predictionsbot/connector/impl/apifootball/model/Status.kt
package at.hrechny.predictionsbot.connector.impl.apifootball.model

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

@JsonIgnoreProperties(ignoreUnknown = true)
class Status {
    @field:JsonProperty("long")
    var statusLong: String? = null

    @field:JsonProperty("short")
    var status: FixtureStatusEnum? = null

    var elapsed: Int? = null
}
